src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -12,12 +12,17 @@
 
 @if (_trace is { } trace)
 {
-    <div class="trace-detail-layout">
-        <div class="trace-detail-header page-header">
-            <h1 style="display:inline-block">@GetResourceName(trace.FirstSpan.Source): @trace.FirstSpan.Name</h1>
-            <span class="trace-id">@OtlpHelpers.ToShortenedId(trace.TraceId)</span>
-        </div>
-        <FluentToolbar Orientation="Orientation.Horizontal">
+    <AspirePageContentLayout
```

Comment:
This is obviously broken:

![image](https://github.com/dotnet/aspire/assets/303201/7c84729f-2694-4b81-99a5-54b45972cb9e)

Reply:
Blazor deterministic styles were breaking even though they should not have - the solution posed [here](https://khalidabuhakmeh.com/blazors-css-isolation-deep-issue-and-solution) worked.